nilearn/interfaces/fmriprep/load_confounds_compcor.py

```diff
@@ -47,7 +79,21 @@ def _select_compcor(compcor_cols, n_compcor):
 
 
 def _check_compcor_method(compcor):
-    """Load compcor options and check if method is acceptable."""
+    """Load compcor options and check if method is acceptable.
+
+    Parameters
+    ----------
+    compcor : str
```

Comment:
```suggestion
    compcor : :obj: `str`
```

Reply:
These functions are not exposed in the documentation. Is it still worth doing it?

Reply:
No, indeed.